ComparetheTriplets.cs

```diff
@@ -0,0 +1,68 @@
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+using System.Text;
+using System;
+
+class Result
+{
+
+    /*
+     * Complete the 'compareTriplets' function below.
+     *
+     * The function is expected to return an INTEGER_ARRAY.
+     * The function accepts following parameters:
+     *  1. INTEGER_ARRAY a
+     *  2. INTEGER_ARRAY b
+     */
+
+    public static List<int> compareTriplets(List<int> a, List<int> b)
+    {
+        Int32 length = a.Count;
+        int alice=0;
+        int bob=0;
+      for(int i=0;i< length;i++)
+      {
+          if(a[i]>b[i])
+          {
+              alice=1+alice;
+          }
+          else if(a[i]<b[i]){
+              bob=1+bob;
+          }
+      }
+      
+      List<int> result = new List<int>();
+result.Add(alice);
+result.Add(bob);
```

Comment:
please maintain proper indentation

Reply:
@narayanagunturi I have fixed the problem stated as stated im comment